pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/MethodTypeResolution.java

```diff
@@ -603,6 +624,10 @@ public static boolean isMethodConvertible(JavaTypeDefinition parameter, JavaType
 
 
     public static boolean isSubtypeable(JavaTypeDefinition parameter, ASTExpression argument) {
+        if (argument.getTypeDefinition() == null) {
+            LOG.log(Level.FINE, "No type information for node {0}", argument.toString());
+            return true;
```

Comment:
is true a good default? shouldn't it be false upon unknown types?

Reply:
I think, I tried it - and got not so good results: In that case, we might not find the correct method call...on the other hand, we might match a wrong method call...

It's also similar to https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/MethodTypeResolution.java#L619 - where we originally got a NPE.

Reply:
sounds reasonable, thanks!